pages/ksphere/konvoy/1.5.0-beta/tutorials/disable-audit-logs/index.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: layout.pug
+navigationTitle: Disable Audit Log Collection
+title: Disable Audit Log Collection
+menuWeight: 11
+excerpt: Audit Logs are collected via the FluentBit Addon and enabled for collection by default. In some cases this may not be necessary and turning off audit-log-collection would be ideal. Here we show the procedure to disable Audit Log Collection.
+enterprise: false
+---
+
+### Disable Audit Log Collection
```

Comment:
I vote to put the contents of this page in `fluentbit/index.md` as it's closest to the addon info. Right now tutorials are not ordered or categorized so you have kind of a mish-mash of random tutorials. And I think I remember docs team suggesting tutorials are (or can be a long) instructional guide to completing a set of tasks. A 1 step tutorial for switching an addon component on off seems a bit light.